layouts/partials/pagelayout.html

```diff
@@ -6,7 +6,7 @@
         <h1 class="primary-color-dark f2">
             Hugo Themes
             {{ if not .context.IsHome -}}
-              <span class="f4 mt2 mt0-l db dib-l">({{ with $.Data.Singular }}{{ . | humanize }}:{{ end }} {{ .context.Title }})</span>
+              <span class="f4 mt2 mt0-l db dib-l">({{ with $.context.Data.Singular }}{{ . | humanize }}:{{ end }} {{ .context.Title }})</span>
```

Comment:
You should also move the space before the end. I'm pretty sure I had that in my example. Now, if the with is false, there will be an extra space, so "( Blog)".